st2api/st2api/config.py

```diff
@@ -28,10 +28,23 @@
 BASE_DIR = os.path.dirname(os.path.abspath(__file__))
 
 
+def parse_args(args=None):
+    CONF(args=args, version=VERSION_STRING)
+
+
+def register_opts():
+    _register_common_opts()
+    _register_app_opts()
+
+
 def _register_common_opts():
     common_config.register_opts()
 
 
+def get_logging_config_path(service='api'):
```

Comment:
Pushed scheduler and re-scheduler decoupling refactor in 602a8c2afa3e15499eb83553ca585560b4b522da.

Re-scheduler had no place to live in `st2actions/scheduler.py` since it was only used by the notifier code.

Code is now a lot more self-sustaining which means it should also be easier to do the bigger refactoring later on.